frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -3,8 +3,9 @@ import { Row, Card, Col, Button } from "react-bootstrap";
 import { useNavigate } from "react-router-dom";
 import { hasRole } from "main/utils/currentUser";
 import { daysSinceTimestamp } from "main/utils/dateUtils";
+import AnnouncementCard from "main/components/Announcement/AnnouncementCard";
 
-export default function CommonsOverview({ commonsPlus, currentUser }) {
+export default function CommonsOverview({ commonsPlus, currentUser, announcement }) {
```

Comment:
Why are you passing in only one announcement?  There could be many announcements!

Reply:
My teammate has helped me fix this. Now in PlayPage.js, all announcements will be extracted instead of the latest one.